docs/cudf/source/user_guide/guide-to-udfs.ipynb

```diff
@@ -7,13 +7,24 @@
     "# Overview of User Defined Functions with cuDF"
```

Comment:
<blockquote>This example is adapted from cuDF's <a href="https://docs.rapids.ai/api/cudf/stable/api.html#cudf.core.dataframe.DataFrame.rolling" target="_blank">API documentation</a>. </blockquote><p> </p><p>Orthogonal to this PR, but the link here is broken. Ditto below with apply_grouped.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/10607/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='75'/>

Reply:
I fixed this link but actually I think the docs for `apply_grouped` are missing as I dont see them anywhere in our stable docs. Looking into this separately.